configure

```diff
@@ -276,41 +278,39 @@ elif (args.eigen != ""):
     f.write("EIGEN = " + args.eigen + "\n")
 
 #
-# Compiler
+# Compiler and MPI library
 #
 if not args.comp in ['g++','icc','clang++']:
     raise Exception(color.red + "ERROR" + color.reset + ": Compiler must be g++, clang++, or icc")
-if (args.comp == "gcc"): f.write("COMP = GCC\n")
-elif (args.comp == "clang"): f.write("COMP = CLANG\n")
-elif (args.comp == "intel"): f.write("COMP = INTEL\n")
-message("Comp-Type", str(args.comp))
-shellmessage("Comp-Version",args.comp+" --version")
-shellmessage("Comp-MPI","mpicxx -cxx="+args.comp+" -show")
-f.write("CC = mpicxx -cxx="+args.comp+"\n")
-if args.comp == 'g++':     
-    f.write("MPI_LIB = -lgfortran -lmpich\n")
+
+if not args.alternate_mpi and not shutil.which("mpichversion"):
+    raise Exception(color.red + "ERROR" + color.reset + ": An MPICH or MVAPICH installation was not found, to compile with another MPI library please use " + code("--alternate-mpi"))
+
+shellmessage("Compiler (" + args.comp + ")", args.comp + " --version")
+shellmessage("MPI Library","mpiexec --version")
```

Comment:
@nmnobre this should be `mpicxx` rather than `mpiexec` or `mpirun`.

Reply:
If you use `mpicxx`, the flag gets passed to the backend compiler, so you get the exact same output as in the previous line.

Reply:
@nmnobre No you do not.
Please check out the `development` branch and run `./configure` with mpich as the default. Then check out your branch and run `./configure`. You will see that the outputs are different.

Reply:
@nmnobre exactly. Please keep the output of the configure script identical to what is output in the development branch.

Reply:
I've reinstated "-v" instead of "--version". I believe we aren't losing any info now.